### PR TITLE
Remove legacy DE card from Dashboard.

### PR DESCRIPTION
**Pull request by sri2k1us**
SSIA

### CODE REVIEW

File: src/components/dashboard/dashboardItem/LegacyDE.js [hunk 1]
```diff
@@ -1,87 +0,0 @@
...
-            <CardHeader
-                avatar={
-                    <NewReleases
-                        style={{ color: theme.palette.gold }}
-                        fontSize="large"
-                    />
-                }
-                title={
-                    <Typography variant="subtitle2">
-                        {t("legacyDeTitle")}
```
Comment (psarando) [inline src/components/dashboard/dashboardItem/LegacyDE.js:47]: This string can be deleted from the `dashboard.json` locales file.

File: src/components/dashboard/dashboardItem/LegacyDE.js [hunk 2]
```diff
@@ -1,87 +0,0 @@
...
-                    </Typography>
-                }
-            />
-            <CardContent>
-                <Typography variant="body2" gutterBottom>
-                    <Trans
-                        t={t}
-                        i18nKey="legacyDeMessage"
```
Comment (psarando) [inline src/components/dashboard/dashboardItem/LegacyDE.js:55]: These strings can be deleted from the `dashboard.json` locales file.

File: src/components/dashboard/dashboardItem/LegacyDE.js [hunk 3]
```diff
@@ -1,87 +0,0 @@
...
-                        tOptions={{
-                            context: userProfile?.id ? "loggedIn" : "loggedOut",
-                        }}
-                        components={{
-                            legacyLink: (
-                                <DELink
-                                    text={t("legacyDeName")}
```
Comment (psarando) [inline src/components/dashboard/dashboardItem/LegacyDE.js:62]: This string can be deleted from the `dashboard.json` locales file.

**Review by psarando — APPROVED** [review]
LGTM 👍

**Comment by sri2k1us** [comment]
Thanks for the reviews.

**Pull request merged**